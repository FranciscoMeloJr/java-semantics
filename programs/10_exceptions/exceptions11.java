class main {

  main(String[] args) {
    try {
      throw new RuntimeException("a");
    } catch (RuntimeException e) {
      System.out.print(e+" ");
    }
    System.out.println(42);
    System.out.println("Done!");
  }
}

public class exceptions11 {
  public static void main(String[] args) {
    new main(args);
  }
}

// 4 42
// Done!
