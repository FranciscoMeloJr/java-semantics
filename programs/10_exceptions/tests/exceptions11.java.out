java.lang.RuntimeException: a 42
Done!
